stochastic_physics/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+
+if(32BIT)
```

Comment:
should this CMakeLists.txt be added to the stochastic_physics repo? I know there is dependencies on the structure of stochastic_physics, but my understanding is that this file should belong to stochastic_physics.

Reply:
It should. Eventually. I just wanted to reduce the number of repositories I have to update in this first cmake commit.